lesson27/js/task_1.js

```diff
@@ -0,0 +1,61 @@
+// 1. Создать массив «Список покупок». Каждый элемент массива является объектом,
+// который содержит название продукта, необходимое количество и куплен или нет.
+// Написать несколько функций для работы с таким массивом.
+
+// Вывод всего списка на экран таким образом, чтобы сначала шли некупленные продукты,
+// а потом – купленные.
+// Добавление покупки в список. Учтите, что при добавлении покупки с уже существующим
+//  в списке продуктом, необходимо увеличивать количество в существующей покупке, а не добавлять новую.
+// Покупка продукта. Функция принимает название продукта и отмечает его как купленный.
+let listOfPurchases = [
+  {
+    product: "Bread",
+    ammount: 2,
+    purchased: false,
+  },
```

Comment:
Об'єкт по суті називається продукт, тому краще тут `name`
```suggestion
  {
    name: "Bread",
    ammount: 2,
    purchased: false,
  },
```